autoload/lsp/options.vim

```diff
@@ -70,6 +70,10 @@ export var lspOptions: dict<any> = {
   # Allowed values: 'above' | 'below' | 'after' (default is 'above')
   diagVirtualTextAlign: 'above',
 
+  # Wrapping of virtual diagnostic text, when showDiagWithVirtualText is true.
+  # Allowed valuse: 'default' | 'truncate' | 'wrap' (default is 'default')
+  diagVirtualTextWrap: 'default',
```

Comment:
Can you update the help text with a description of this new option and also the README file?

Reply:
Sure. I'll post here once I've got that finished.

Reply:
I've committed the documentation changes to my fork.